src/main/java/cz/cvut/kbss/analysis/model/Item.java

```diff
@@ -0,0 +1,45 @@
+package cz.cvut.kbss.analysis.model;
+
+import cz.cvut.kbss.analysis.util.Vocabulary;
+import cz.cvut.kbss.jopa.model.annotations.*;
+import lombok.Getter;
+import lombok.Setter;
+
+import java.util.HashSet;
+import java.util.Set;
+
+@OWLClass(iri = "http://onto.fel.cvut.cz/ontologies/fta-fmea-application/item")
```

Comment:
Why do we have this outside of ontology?

Reply:
Fixed